src/justfile.rs

```diff
@@ -724,6 +724,54 @@ mod tests {
     }
   }
 
+  run_error! {
```

Comment:
I'm trying to move away from the test generation macros. New tests should be integration tests, and use `Test::new()`, see examples in the `tests` directory.